### ISSUE TITLE
Dialog: Ensures every ARIA dialog and alertdialog node has an accessible name.

### ISSUE THREAD

**Issue by WTW-Steve-Martin** [primefaces/primeng#14950]
### Describe the bug

The Axe Dev Tools for Accessibility is reporting the following issue:
![image](https://github.com/primefaces/primeng/assets/153538315/169b170a-127f-41d7-8857-a5dcbc9b2928)

Axe is highlighting that the dialog has a missing aria-label.

Please could we update this control so that the Axe scan passes?

### Environment

Windows 10

### Angular version

17.2.3

### PrimeNG version

17.9.0

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

18.18.0

### Steps to reproduce the behavior

1. Install the Axe Dev Tools for Accessibility plug-in.
2. Visit the PrimeNG dialog page (https://primeng.org/dialog) on Chrome/Edge browser.
3. Scan the page with Axe

### Expected behavior

The Axe scan doesn't return this error.
